src/tools/pub/cmd/pub-get.md

```diff
@@ -143,6 +143,17 @@ command-line argument, as mentioned above.
 For options that apply to all pub commands, see
 [Global options](/tools/pub/cmd#global-options).
 
+### `--dry-run` or `-n`
```

Comment:
Interesting approach! At first it seemed weird to use h3s (so big, so emphatic), but I like having the options visible in the TOC.

The only badness is that the code font doesn't look great in headings (because of the color maybe? the huge size?) and isn't picked up in the TOC. Still, we could change the code font color/size in h3s to mitigate this.

@parlough, do you have an opinion on making h3s the standard way to document options for tools? Currently we use `<dl>`s instead (e.g. at https://dart.dev/tools/pub/cmd/pub-cache).

The only place I've found where it wouldn't work is in http://localhost:4000/tools/dart2js#options, which has so many options that they're divided into groups. (Weirdly, that page uses h4s instead of h3s, so even the groups don't appear in the TOC.) But I think that exception is fine.

Reply:
@kwalrath I think using h3s seems like a fair idea, for both being present in the TOC but also to simplify contributing and later editing.

Do you have anything in particular in mind in terms of the style change you mentioned? It does seem slightly strange to mix code style with normal text in headers.

Reply:
I used h3's primarily because I wanted to keep consistent with other pages that listed options ( my basis was https://dart.dev/tools/pub/cmd/pub-lish ).